Update documentation for Invite User and Access Management permissions (#9171)
DataDog/documentation: content/en/account_management/authn_mapping/_index.md

  text: "RBAC for Log Management"
---

> If you are using Federated Authentication mechanisms, this API allows you to automatically map groups of users to roles in Datadog using attributes sent from your Identity Provider. Users will need to use an Application Key owned by someone with the Access Management permission in order to create and manage Authentication Mappings via the API.

Reviewer: ```suggestion
If you are using Federated Authentication mechanisms, this API allows you to automatically map groups of users to roles in Datadog using attributes sent from your Identity Provider. To create and manage Authentication Mappings through the API, users need to use an application key owned by someone with the Access Management permission.
```
